packages/app/src/systems/Core/components/Layout/Layout.tsx

```diff
@@ -124,6 +124,7 @@ export const styles = {
     height: WALLET_HEIGHT,
     background:
       'linear-gradient(210.43deg, #0E221B 0%, #071614 10.03%, #0C0E0D 18.38%)',
+    zIndex: '$0',
```

Comment:
Why we need this here?

Reply:
Since the dropdown has the `zIndex: '$1'`, to avoid it being present on top of the overlay content (refer to below screenshot), setting the layout to be at `zIndex: '$0'`

<img width="348" alt="network-dropdown" src="https://user-images.githubusercontent.com/5172086/218546301-f5f332fa-490a-4c13-815a-e5dacbe3eea5.png">